pages/common/multitail.md

```diff
@@ -0,0 +1,23 @@
+# multitail
+
+> Extension of tail.
+
+- Tail all files matching a pattern in a single stream:
+
+`multitail -Q 1 '{{pattern}}'`
+
+- Tail all files in a directory in a single stream:
+
+`multitail -Q 1 '{{directory}}/*'`
+
+- Automatically add new files to a window:
+
+`multitail -Q {{pattern}}`
+
+- Show 5 logfiles while merging 2 and put them in 2 columns with only one in the left column:
+
+`multitail -s 2 -sn 1,3 {{mergefile}} -I {{file1}} {{file2}} {{file3}} {{file4}}`
+
+- More examples:
```

Comment:
Ah, I see. The link in this example should be the one in the more information I've mentioned in the comment above I think.